docs/internals/security.txt

```diff
@@ -38,6 +38,38 @@ action to be taken, you may receive further followup emails.
 
 .. _our public Trac instance: https://code.djangoproject.com/query
 
+.. _reporting-guidelines:
+
+Reporting guidelines
+====================
+
+An identified vulnerability should meet the following guidelines to be
```

Comment:
I would suggest this slight rewording because, to me, it felt a bit a contradiction to first think on "identified" and then "oh is not fully identified":
```suggestion
A potential vulnerability should meet the following guidelines to be
```